CHANGELOG.md

```diff
@@ -1,3 +1,18 @@
+# dbt_twilio_source v0.1.1
+
+## Features
+- Added the ability to disable models related to the `CALL` source table. Refer to the [README](https://github.com/fivetran/dbt_twilio_source?tab=readme-ov-file#step-4-enablingdisabling-models) for more details ([PR #5](https://github.com/fivetran/dbt_twilio_source/pull/5)).
+
+## Under the Hood
+- Adjusted the case of loader name in in `src_twilio.yml` ([PR #5](https://github.com/fivetran/dbt_twilio_source/pull/5)).
```

Comment:
Can you mention here what the adjustment was. Just to clearly define the change from fivetran to Fivetran

Reply:
updated by raphael